src/lib/database/schema.ts

```diff
@@ -1,16 +1,21 @@
+import type { DeviceModelV1, Manufacturer } from "@seamapi/types/devicedb"
 import type { HoistedStoreApi } from "zustand-hoist"
 
-import type { Thing, ThingInitializer } from "lib/models.ts"
+import type { StoredDeviceModelV1, StoredManufacturer } from "lib/models.ts"
 
 export interface DatabaseState {
-  _id: number
-  things: Thing[]
+  manufacturers: StoredManufacturer[]
+  device_models: StoredDeviceModelV1[]
   vercel_protection_secret: string
+  live_seam_connect_endpoint: string | null
```

Comment:
Calling this `live_seam_connect_endpoint` creates a dependency / requires knowledge of seam connect / assumes a particular infa. Can we call this `image_proxy_origin` or `image_proxy_endpoint` instead?

Reply:
I think it's actually correct to assume that only seam connect can be used here since it's a pretty specific proxy method

Reply:
What is specific about this proxy method?

Reply:
I started writing an argument for this and then realized I don't really care that much haha

can refactor

Reply:
c222b5f